lib/tests/streamlit/server_test.py

```diff
@@ -85,36 +85,36 @@ def tearDown(self):
         return super().tearDown()
 
     @tornado.testing.gen_test
-    def test_start_stop(self):
+    async def test_start_stop(self):
```

Comment:
Are the `@tornado.testing.gen_test` decorators still needed here? They were removed in `app_session_test.py`, but I'm not sure if something about the way that server tests are written require the decorator to still be around

Reply:
Yeah - this is unfortunately still required for tornado async tests. If you try to perform an async operation in a Tornado-based test without this decorator, it'll fail (and tell you to add the decorator).

(The AppSession tests no longer use tornado, which is why the decorators were removed there.)

I haven't looked into it, but I think the decorator is probably required because Tornado supports both new-style async/await coroutine syntax _and_ its old generator `yield`-based syntax.